src/passes/storageAllocator.ts

```diff
@@ -26,14 +27,16 @@ export class StorageAllocator extends ASTMapper {
     let usedStorage = 0;
     const allocations: Map<VariableDeclaration, number> = new Map();
     node.vStateVariables.forEach((v) => {
-      const width = CairoType.fromSol(
-        getNodeType(v, ast.compilerVersion),
-        ast,
-        TypeConversionContext.StorageAllocation,
-      ).width;
-      allocations.set(v, usedStorage);
-      usedStorage += width;
-      extractInitialisation(v, initialisationBlock, ast);
+      if (v.mutability !== Mutability.Constant) {
```

Comment:
I think it would be safer to use `!isCairoConstant` for the condition here, to ensure that no variables are both ignored here and later on